package.json

```diff
@@ -125,37 +125,65 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.12.1",
-    "@zwave-js/server": "^1.2.0",
+    "@release-it/conventional-changelog": "^2.0.1",
+    "@types/node": "^14.14.34",
```

Comment:
@raman325 please remove all these changes except `@zwave-js/server`. Those belong in `devDependencies`

Reply:
thanks, looks like I was running the wrong `node` and `npm`